_posts/2024-02-15-OpenSearchs-2023-Wrapped.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "OpenSearch Project's 2023 wrapped: a year in review"
+authors:
+  - dtaivpp
+date: 2024-02-14
+categories:
+  - community
+meta_keywords: OpenSearch 2023, year in review, new in opensearch, observability features, 2023 recap, 2024 plan
```

Comment:
Suggested meta updates: 
**meta_keywords:** OpenSearch Project year in review, OpenSearch user experience, OpenSearch Project roadmap

**meta_description:** Discover how the OpenSearch Project plans to collaborate, innovate, and drive the project forward in 2024 while taking a look back project growth and releases in 2023.

Reply:
Done and done! Thank you